alacritty_terminal/src/tty/windows/child.rs

```diff
@@ -29,15 +31,15 @@ impl ChildExitWatcher {
     pub fn new(child_handle: HANDLE) -> Result<ChildExitWatcher, Error> {
         let (event_tx, event_rx) = channel::<ChildEvent>();
 
-        let mut wait_handle: HANDLE = 0 as HANDLE;
+        let mut wait_handle = 0isize;
```

Comment:
Could you use proper type alias? There's `HANDLE` in foundation.

Reply:
Yes, I'll do it this way, it's better.